django/db/models/fields/__init__.py

```diff
@@ -2409,7 +2422,7 @@ def get_internal_type(self):
         return 'BigAutoField'
 
     def rel_db_type(self, connection):
-        return BigIntegerField().db_type(connection=connection)
+        return PositiveBigIntegerField().db_type(connection=connection)
```

Comment:
I'm afraid that's backward incompatible, this should be handled in #11900.

Reply:
Yup. This is where we'd need to define `PositiveBigAutoField` which would have this set to `PositiveBigIntegerField`.

Reply:
Agreed!